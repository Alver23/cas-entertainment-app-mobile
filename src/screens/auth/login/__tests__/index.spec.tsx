// Dependencies
import React from 'react';
import { shallow, ShallowRendererProps } from 'enzyme';

// Screens
import { LoginScreen } from '../index';

// Mocks
jest.mock('@containers/login-form', () => ({ LoginForm: jest.fn() }));

describe('LoginScreen', () => {
  let component: ShallowRendererProps;

  beforeEach(() => {
    component = shallow(<LoginScreen />);
  });

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});
