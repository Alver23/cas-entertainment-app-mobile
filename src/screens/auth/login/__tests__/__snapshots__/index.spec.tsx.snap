// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginScreen should save a snapshot of the component 1`] = `
<LayoutView
  style={
    Object {
      "justifyContent": "center",
    }
  }
>
  <Memo(Component) />
  <mockConstructor />
</LayoutView>
`;
